configure.ac

```diff
@@ -1,7 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.61)
-AC_INIT([lxcfs], [3.0.0], [lxc-devel@lists.linuxcontainers.org])
+AC_INIT([lxcfs], [3.0.3], [lxc-devel@lists.linuxcontainers.org])
```

Comment:
@stgraber is this version number OK to change in the code, or is it generated in some other process? I noticed it was a couple of minor releases out and wondered if that was intentional?

Reply:
@tomponline this shouldn't be changed in master.

The point releases versions are only committed to the stable-3.0 branch.
Next time we release what's in master, it should be 3.1.0 and we'll change configure.ac then.

Reply:
If it matters, we could do the same thing we do in LXC where we have a DEVEL flag in configure.ac to indicate that this is the development branch pre-release, leading to all tarballs and version strings including a -devel suffix.

Reply:
@stgraber I understand. I needed it to test the RPM SPEC file in this PR, as I was testing it being built against the 3.0.3 source tarball. However I can change it back now I can see its working OK.